routes/table.js

```diff
@@ -0,0 +1,78 @@
+var express = require('express');
+var router = express.Router();
+var Table = require('../models/table');
+var tableMapper = require('../mappers/tableMapper');
+var eventMapper = require('../mappers/eventMapper');
+var isLoggedIn = require('../config/utils').isLoggedIn;
+var isAdminUser = require('../config/utils').isAdminUser;
+
+router.get('/:event_id',isLoggedIn, isAdminUser, function(req, res) {
+	eventMapper.findEventBy_event_id(req.params.event_id,function(err,result){
+
+		var eventName = result.title;
+
+		tableMapper.findTablesByEventId(req.params.event_id,function(err,table){
+			res.render('table', {table:table,event_id:req.params.event_id,eventName:eventName, succ: req.flash('succ')});
+		});	
+	});	
+});
+
+router.post('/edit/:event_id/:tableNumber/:labelAmount',isLoggedIn, isAdminUser, function(req, res) {
+	
+	var Labels = [];
+	var Seats = [];
+	var data = req.body;
+	for(var item in data){
+		if(item.split('-')[0] == 'label'){
+			Labels.push(req.body[item]);
+		}
+		else{
+			Seats.push(req.body[item]);
+		}
+	}
+	var tableObj = new Table({tableNumber:req.params.tableNumber,tableLabels:Labels,size:req.params.labelAmount,seatLabels:Seats,eventId:req.params.event_id});
```

Comment:
Your labelAmount param is always 0 for me, this means the size gets reset in DB to 0 everytime you edit the table in any way